recipes/connectionpool/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ConnectionPool" %}
```

Comment:
If you follow the example meta.yaml (https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml), this should look something like:

```
{% set name = "ConnectionPool" %}
{% set version = "0.1.0" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
    url: https://github.com/jmeppley/{{ name }}/archive/v{{ version }}.tar.gz
    sha256: 123861f0ff0b7d0967d2fb3cbefd13a461091be7a5451942f820c7f17062669f"

build:
  number: 0
  noarch: python
  script: "{{ PYTHON }} -m pip install . -vv"

requirements:
  host:
    - python
    - pip
  run:
    - python

about:
  home: https://github.com/jmeppley/ConnectionPool
  license: MIT
  license_family: MIT
  license_file: LICENSE
  summary: 'A simple thread-safe pure-python connection pool'

extra:
  recipe-maintainers:
    - jmeppley
```

Reply:
From the looks of it, we overlapped for a year at HC. I was '98.